src/Autofac.Integration.AspNetCore.Multitenant/AutofacMultitenantServiceProviderFactory.cs

```diff
@@ -65,6 +66,17 @@ public IServiceProvider CreateServiceProvider(ContainerBuilder containerBuilder)
               .AsSelf()
               .ExternallyOwned();
 
+            containerBuilder.Register<ServiceScopeFactory>(_ =>
+            {
+                return scope =>
```

Comment:
NICE! Great work with this. Honestly just a tiny bit surprised that it works, but I'm not going to question success. 😆